config/routes.rb

```diff
@@ -42,6 +42,11 @@
         get '/find_all', to: 'search#index'
         get '/random', to: "random#show"
         get '/:id', to: 'invoices#show'
+        get '/:id/transactions', to: "invoice_transactions#show"
+        get '/:id/invoice_items', to: "invoice_invoice_items#show"
+        get '/:id/items', to: "invoice_items#show"
+        get '/:id/customer', to: "invoice_customer#show"
+        get '/:id/merchant', to: "invoice_merchant#index"
       end
```

Comment:
@Tyjoo26 These routes are a little inconsistent. We have transactions, invoice_items, and items combined with a show, and merchant combined with an index